docs/how-to/how-to-set-sandbox-config-kata.md

```diff
@@ -80,6 +82,7 @@ There are several kinds of Kata configurations and they are listed below.
 | `io.katacontainers.config.hypervisor.kernel` | string | the kernel used to boot the container VM |
 | `io.katacontainers.config.hypervisor.machine_accelerators` | string | machine specific accelerators for the hypervisor |
 | `io.katacontainers.config.hypervisor.machine_type` | string | the type of machine being emulated by the hypervisor |
+| `io.katacontainers.config.hypervisor.remote_machine_type` | string | the type of machine to be created by the remote hypervisor |
```

Comment:
I think we probably should rename this to something like remote_instance_type or similar. Machine type in kata containers means something different, so now we've decoupled I think we should be clearer for the remote hypervisor annotations

Reply:
Also - can you add some line break to you commit message - we need have max line length of 150 chars and more like 80ish is more common. Thanks!

Reply:
Thanks Steve, will incorporate them..

Reply:
Thanks Steve for the suggestions, will incorporate these..

Reply:
Marking this comment as resolved as we are no more pursuing the solution via this route.